### ISSUE TITLE
go-groups does not respect comments in import block

### ISSUE THREAD

**Issue by mcavoyk** [indeedeng/go-groups#11]
go version: 1.14.5
go-groups version: 0.0.1

```golang
package main

import (
        // example comment
	"fmt"
)

func main() {
	fmt.Println("Hello world")
}
```

`goimports` leaves comment while `go-groups` removes.

**Comment by mcavoyk** [comment]
May be related to https://github.com/indeedeng/go-groups/issues/10

**Comment by cnmcavoy** [comment]
Fixed in v1.1.0
